jobs/nginx/atlas.hcl

```diff
@@ -0,0 +1,49 @@
+job "atlas" {
+  datacenters = ["aperture"]
+  type = "service"
+
```

Comment:
Can we add a `meta` field here for the git-sha? That'll allow us to add our ci later